astropy/io/ascii/tests/test_c_reader.py

```diff
@@ -1481,3 +1481,19 @@ def test_conversion_fast(fast_reader):
     assert table['F'].dtype.kind in ('S', 'U')
     assert table['G'].dtype.kind in ('S', 'U')
     assert table['H'].dtype.kind in ('S', 'U')
+
+
+@pytest.mark.parametrize('delimiter', ['\n', '\r'])
```

Comment:
Is the Windows style `'\r\n' also a supported use case?

Reply:
`delimiter` can only be a single character, so CR+LF is not an option, but for the parsed file any combination of `\n`, `\r` and `\r\n` is accepted as linebreak. Could add that to the test `text`.